src/index.ts

```diff
@@ -0,0 +1,22 @@
+import dotenv from 'dotenv';
+import app from './app';
+import models from './db/models/index';
+
+dotenv.config();
+
+const PORT = process.env.PORT || 9000;
+
+models.sequelize.sync().then(() => {
+  app.on('error', (error) => {
+    // tslint:disable-next-line:no-console
+    console.log(error);
+  });
+  app.on('listening', (listen) => {
+    // tslint:disable-next-line:no-console
+    console.log(listen);
+  });
+  app.listen(PORT, () => {
+    // tslint:disable-next-line:no-console
```

Comment:
Instead of duplicating this ` tslint:disable-` why not have one at the beginning for the whole file?

Reply:
Done